doc/tutorials/rt_linux.rst

```diff
@@ -53,17 +53,16 @@ system on Intel KBL NUC with a SATA SSD as ``/dev/sda`` and an NVME SSD as
 
    a. Download Linux kernel real-time patch::
 
-         $ wget https://mirrors.edge.kernel.org/pub/linux/kernel/projects/rt/4.19/patch-4.19.15-rt12.patch.xz
+         $ wget https://mirrors.edge.kernel.org/pub/linux/kernel/projects/rt/4.19/patch-4.19.31-rt18.patch.xz
 
-   #. Sync the kernel code to acrn-2019w04.5-150000p::
+   #. Sync the kernel code to acrn-2019w17.4-160000p::
 
          $ git clone https://github.com/projectacrn/acrn-kernel.git
-         $ git checkout acrn-2019w04.5-150000p
+         $ git checkout acrn-2019w17.4-160000p
          $ cd acrn-kernel
-         $ xzcat ../patch-4.19.15-rt12.patch.xz | patch -p1
-
-      .. note:: This patch will report one conflict with ``base.c`` that
-         you can ignore.
+         $ wget https://github.com/projectacrn/acrn-hypervisor/blob/master/doc/tutorials/rt_linux.patch
```

Comment:
I'm pretty sure this will actually make you download an HTML page, what we most likely need instead is this:

```suggestion
         $ wget https://raw.githubusercontent.com/projectacrn/acrn-hypervisor/master/doc/tutorials/rt_linux.patch
```

Reply:
Good catch.  Yes, raw.githubusercontent.com is the place to aim wget.

Reply:
I tested it before commit, for example, I downloaded the patch file in ubuntu by "wget https://github.com/lijianpi/acrn-hypervisor/blob/master/doc/tutorials/rt_linux.patch".

Reply:
While the wget indeed works with this github.com URL, the resulting file is an HTML file and not the plain text raw patch file.  I see this if I do a ``wget https://github.com/lijianpi/acrn-hypervisor/blob/master/doc/tutorials/rt_linux.patch`` :
```

<!DOCTYPE html>
<html lang="en">
  <head>
    <meta charset="utf-8">
  <link rel="dns-prefetch" href="https://github.githubassets.com">
  <link rel="dns-prefetch" href="https://avatars0.githubusercontent.com">
  <link rel="dns-prefetch" href="https://avatars1.githubusercontent.com">
  <link rel="dns-prefetch" href="https://avatars2.githubusercontent.com">
  <link rel="dns-prefetch" href="https://avatars3.githubusercontent.com">
  <link rel="dns-prefetch" href="https://github-cloud.s3.amazonaws.com">
  <link rel="dns-prefetch" href="https://user-images.githubusercontent.com/">



  <link crossorigin="anonymous" media="all" integrity="sha512-iwkZZWcsyMNnn/6c9v2ab3e7h4ZiFTOEZ9R6jj75bN9JzYuYjQ/AC7ufEcfpqd4GcTw
cM+p2OhPzRSyeKL7NMQ==" rel="stylesheet" href="https://github.githubassets.com/assets/frameworks-f331a618befc2bf99be870c538f9ac95.
css" />
  <link crossorigin="anonymous" media="all" integrity="sha512-cNSmDJd2vJZ5aluOBlLP6oTxa7tvVIc54LZk60zATiwTzs+aicoVb3r8TgU/1eiz2s6
Nd4yPb7b1LxilONyJOg==" rel="stylesheet" href="https://github.githubassets.com/assets/site-7d7d47ab5852bb08d1180249a6c36bf5.css" /
>
    <link crossorigin="anonymous" media="all" integrity="sha512-7xMIa/sc0b/JOK9wvEgKXxcxFgWk2iqJgQOWB4tqu1N+35igEY2AHI0le6Xg+d3e1
d7aGtPfsBJV0qzBgpWeYA==" rel="stylesheet" href="https://github.githubassets.com/assets/github-d77687b19cac549d2cbe4c45e71e955a.cs
s" />

. . .
```